src/pages/docs/game/chat.mdx

```diff
@@ -0,0 +1,20 @@
+---
+layout: ../../../layouts/docs.astro
+---
+
+
+# __Chat__ 
+[@veritanuda](https://keybase.io/veritanuda)
+
+In game chat is now possible by simply clicking the `Chat` button in the bottom left of the viewport window.
+
+![Chat button](assets/chat_button_default.png)
+
+When a chat message is sent by any user then a number of unread messages will appear next to the button.
+
+![Chat Button Messages waiting](assets/chat_button_waiting.png)
+
+The chat window accepts Markdown input meaning styles and links can be included as well as more advanced formatting. See the [Markdown Tutorial](../../../learn/dm/Markdown_Tutorial/markdown)
+
+The chat is ***ephemeral*** meaning the current chat session will only exist in that context window. Should for any reason you refresh the page the chat history will be lost to you.
+
```

Comment:
I would also mention that the chat functionality can be disabled by the DM in the DM Settings (the new feature tab).

Reply:
That is a new feature after the last update so yeah. I can add that. :)